collector.go

```diff
@@ -67,6 +74,29 @@ var (
 			"sv_tested":  {GAUGE, "server_testing_connections", 1, "Server connections currently running either server_reset_query or server_check_query, shown as connection"},
 			"sv_login":   {GAUGE, "server_login_connections", 1, "Server connections currently in the process of logging in, shown as connection"},
 			"maxwait":    {GAUGE, "client_maxwait_seconds", 1, "Age of oldest unserved client connection, shown as second"},
+			"pool_mode":  {LABEL, "pool_mode", 1, "The pooling mode in use."},
+		},
+		"lists": {
+			"databases":     {GAUGE, "databases", 1, "Number of databases"},
+			"users":         {GAUGE, "users", 1, "Number of users"},
+			"pools":         {GAUGE, "pools", 1, "Number of pools"},
+			"free_clients":  {GAUGE, "free_clients", 1, "Number of free clients"},
+			"used_clients":  {GAUGE, "used_clients", 1, "Number of used clients"},
+			"login_clients": {GAUGE, "login_clients", 1, "Number of clients in login state"},
+			"free_servers":  {GAUGE, "free_servers", 1, "Number of free servers"},
+			"used_servers":  {GAUGE, "used_servers", 1, "Number of used servers"},
+			"dns_names":     {GAUGE, "dns_names", 1, "dns names"},
+			"dns_zones":     {GAUGE, "dns_zones", 1, "dns zones"},
+			"dns_queries":   {GAUGE, "dns_queries", 1, "dns queries"},
+			"dns_pending":   {GAUGE, "dns_pending", 1, "dns pending"},
+		},
+
+		"mem": {
```

Comment:
Perhaps the names of these labels should follow https://prometheus.io/docs/practices/naming/?

Reply:
Thanks, I think the documentation needs to be updated too.

@SuperQ Any other comments about naming here?

Reply:
@stanhu I already update the names on the readme
Do you need me to update any other documents?

Reply:
@stanhu & @SuperQ any updates related to this PR